scripts/booklist.js

```diff
@@ -19,6 +19,12 @@ function populateBooks(url){
         }
       }
     }
+  }).fail( function( xhr, status ) {
+    $(".loader").hide();
+
+    $("#resultsTray").css("grid-template-columns", "none").append(
+      $("<p></p>").text(getErrorMessage(status))
```

Comment:
This is unnecessary complex, I think that you only need something like:
```
$('#resultsTray').append(
  $('div').addClass('text-center').html(getErrorMessage(status))
)
```

Reply:
Idea: maybe `xhr` has more info on the type of error than the `status`. Please check its attributes.

Reply:
Your solution does not work as intended, because #resultsTray is a grid.  This solution might work if I refactor the css to use flexbox or bootstrap classes, but for now I need to disable the grid to center the text.  However, I do like using a <div> instead of a <p>, as it puts in in a nice box which looks like the archive error message